frontend/src/Student/containers/Portfolios.js

```diff
@@ -2,7 +2,8 @@ import { graphql } from 'react-apollo'
 import { connect } from 'react-redux'
```

Comment:
Queries BE for both a students existing portfolios, or the open period they don't have a portfolio on yet.